coordinator/store/stdstore/stdstore.go

```diff
@@ -199,35 +203,10 @@ func (s *StdStore) commit(data map[string][]byte) error {
 
 	s.mux.Lock()
 	defer s.mux.Unlock()
-	if !s.recoveryMode {
+	if !s.recoveryMode && s.sealMode != seal.ModeDisabled {
 		sealedData, err := s.sealer.Seal(s.recoveryData, dataRaw)
 		if err != nil {
-			if !errors.Is(err, seal.ErrMissingEncryptionKey) {
```

Comment:
This introduces a change where we no longer write data to disk before a manifest is set.
I prefer the new behavior, but we might want to mention this in the changelog